docs/reference/search/suggesters.asciidoc

```diff
@@ -116,163 +123,31 @@ and applies to the `my-suggest-1` and `my-suggest-2` suggestions.
 
 [source,js]
 --------------------------------------------------
-"suggest" : {
-  "text" : "the amsterdma meetpu",
+POST _suggest
+{
+  "text" : "tring out Elasticsearch",
   "my-suggest-1" : {
     "term" : {
-      "field" : "title"
+      "field" : "message"
     }
   },
   "my-suggest-2" : {
     "term" : {
-      "field" : "body"
+      "field" : "user"
     }
   }
 }
 --------------------------------------------------
+// CONSOLE
 
 The suggest text can in the above example also be specified as
 suggestion specific option. The suggest text specified on suggestion
 level override the suggest text on the global level.
 
-[float]
-=== Other suggest example
-
-In the below example we request suggestions for the following suggest
```

Comment:
I removed this example because it didn't seem to buy much after the first example.